lib/pure/times.nim

```diff
@@ -1116,7 +1127,7 @@ when not defined(JS):
         gmtoff {.importc: "tm_gmtoff".}: clong
   else:
     type
-      StructTM {.importc: "struct tm", final.} = object
+      StructTM {.importc: "struct tm", pure, final.} = object
```

Comment:
oops, looks like I was wrong. was looking at an inheritable object which obviously cannot be used across a c abi. by the same vein, `final` looks redundant - it's set implicitly - the manuals really hard to follow on these subtle points.